src/index.js

```diff
@@ -1,3 +1,3 @@
-import FetchLoader from './FetchLoader.js'
+import GeneSearchForm from './GeneSearchForm.js'
```

Comment:
Why was this changed? Without `FetchLoader` the component won’t be able to download the data it needs.